packages/agoric-cli/src/main.js

```diff
@@ -19,7 +19,7 @@ import followMain from './follow.js';
 import walletMain from './open.js';
 import { makeWalletCommand } from './commands/wallet.js';
 
-const DEFAULT_DAPP_TEMPLATE = 'dapp-fungible-faucet';
+const DEFAULT_DAPP_TEMPLATE = 'dapp-offer-up';
```

Comment:
I expect the integration tests would need to be updated as well.

Reply:
"the integration tests"?

hmm.. I see https://github.com/Agoric/agoric-sdk/blob/master/.github/workflows/integration.yml#L39
programming in yaml... I'm struggling to follow... seems to lead to...
https://github.com/Agoric/agoric-sdk/blob/master/scripts/registry.sh

But I don't see any use of `yarn create @agoric/dapp`. What am I missing?

Reply:
https://github.com/Agoric/agoric-sdk/blob/d61be8d7fbda7dd3846cba38f8829e36bfae92a4/.github/workflows/integration.yml#L97

Leads to

https://github.com/Agoric/agoric-sdk/blob/d61be8d7fbda7dd3846cba38f8829e36bfae92a4/scripts/registry.sh#L110

The current getting started steps are captured in that script and another.

While neither reference the new dapp-create, that package is just a wrapper for the agoric cli.

Reply:
another? what other? Perhaps all will become clear if I study some ci logs.

Reply:
Yes, `registry.sh` triggers a yarn command in `agoric-cli` after setting up some environment variables
https://github.com/Agoric/agoric-sdk/blob/d61be8d7fbda7dd3846cba38f8829e36bfae92a4/scripts/registry.sh#L117

which ultimately bottoms out in https://github.com/Agoric/agoric-sdk/blob/d61be8d7fbda7dd3846cba38f8829e36bfae92a4/packages/agoric-cli/tools/getting-started.js